javascript/javascript2/week1/spiritanimal.js

```diff
@@ -0,0 +1,28 @@
+const spiritAnimas = [ "fullmoonwolf","The Butterfly","Teen wolf","Spidersense"]
```

Comment:
Typo here 'spiritAnimas' gives an error in the console: `spiritanimal.js:11 Uncaught ReferenceError: spiritAnimals is not defined`